gnovm/stdlibs/time/format.gno

```diff
@@ -538,9 +538,11 @@ func (t Time) GoString() string {
 	buf = append(buf, ", "...)
 	buf = appendInt(buf, t.Nanosecond(), 0)
 	buf = append(buf, ", "...)
-	switch loc := ChainTZ; loc {
+	switch loc := t.Location(); loc {
 	case UTC, nil:
 		buf = append(buf, "time.UTC"...)
+	case ChainTZ:
+		buf = append(buf, "time.ChainTZ"...)
```

Comment:
not sure we want this, it was time.Local... before

Reply:
It's OK

Reply:
I think we should just keep Local. Is there any problem justifying that we want to create a discrepancy with go?

Reply:
IMO, Is it about that local was based on the user local system implying it could be error-prone or unstable.
Now we inject the timezone data.

So do you consider keeping the naming ``Local`` but just modify how it's fetch or you consider the PR should just not modify anything related to the timezone ?